### PR TITLE
Edit signing images

### PR DESCRIPTION
**Pull request by bermudezmt**

### CODE REVIEW

File: ee/dtr/user/manage-images/sign-images/index.md [hunk 1]
```diff
@@ -7,53 +7,52 @@ redirect_from:
...
-Signer. These 2 containers give us the required components to use Docker Content
-Trust right out of the box. [Docker Content
-Trust](/engine/security/trust/content_trust/) allows us to sign image tags,
+Two key components of the Docker Trusted Registry are the Notary Server and the Notary
+Signer. These two containers offer the required components for using Docker Content
+Trust (DCT) out of the box. [Docker Content
+Trust](/engine/security/trust/content_trust/) allows you to sign image tags,
 therefore whoever pulls the image can validate that they are getting the image
-you create, or a forged one. 
+you created.
```
Comment (bermudezmt) [inline ee/dtr/user/manage-images/sign-images/index.md]: @ollypom I'm sorry my edits are delayed on this one, but I couldn't quite make sense of "a forged one" in this sentence. Let me know and I can rephrase.

Comment (ollypom) [reply ee/dtr/user/manage-images/sign-images/index.md]: Hmm, I was referred v poorly to a untrusted / manipulated image.

File: ee/dtr/user/manage-images/sign-images/index.md [hunk 2]
```diff
@@ -7,53 +7,52 @@ redirect_from:
 - /ee/dtr/user/manage-images/sign-images/manage-trusted-repositories/
 ---
 
-2 Key components of the Docker Trusted Registry are the Notary Server and Notary
-Signer. These 2 containers give us the required components to use Docker Content
-Trust right out of the box. [Docker Content
-Trust](/engine/security/trust/content_trust/) allows us to sign image tags,
+Two key components of the Docker Trusted Registry are the Notary Server and the Notary
+Signer. These two containers offer the required components for using Docker Content
```
Comment (ollypom) [inline ee/dtr/user/manage-images/sign-images/index.md]: Nit. I would say something like `provide` not `offer`.

File: ee/dtr/user/manage-images/sign-images/index.md [hunk 3]
```diff
@@ -7,53 +7,52 @@ redirect_from:
...
 UCP has a feature which will prevent [untrusted
 images](/ee/ucp/admin/configure/run-only-the-images-you-trust/) from being
-deployed on the cluster. To use this feature, we first need to upload and sign
-images into DTR. To tie the signed images back to UCP, we will actually sign the
-images with private keys of UCP users. Inside of a UCP Client bundle the
-`key.pem` can be used a User's private key, with the `cert.pem` being a public
-key within a x509 certificate. 
+deployed on the cluster. To use the feature, you need to sign and push images to your DTR. 
+To tie the signed images back to UCP, you need to sign the
+images with the private keys of the UCP users. From a UCP client bundle, you can use
```
Comment (ollypom) [inline ee/dtr/user/manage-images/sign-images/index.md]: Ahh this is my mistake to, but it needs to be more forceful, instead of "you can use" should be "you need to use the key.pem as the private key... etc." Otherwise the UCP feature won't work :S

File: ee/dtr/user/manage-images/sign-images/index.md [hunk 4]
```diff
@@ -63,13 +62,13 @@ Repeat passphrase for new jeff key with ID a453196:
...
+### Initialize the trust metadata and add the user's public certificate
 
-Next, we need to initiate trust metadata for a DTR repository. If you have not
+Next,initiate trust metadata for a DTR repository. If you have not
 done so already, navigate to the **DTR web UI**, and create a repository for
-your image. In this example we've created the `prod/nginx` repository.
+your image. This example uses the `nginx` repository in the `prod` namespace.
 
-As part of initiating the repository, we will add the public key of the UCP User
+As part of initiating the repository, add the public key of the UCP user
```
Comment (ollypom) [inline ee/dtr/user/manage-images/sign-images/index.md]: Nit. I would reword slightly "As part of initiating the repository, the public key of the UCP user will need to be added to the Notary server for this repository as a signer."

**Comment by bermudezmt** [comment]
Thanks, @ollypom
